lock/src/main/java/org/infinispan/lock/configuration/Reliability.java

```diff
@@ -0,0 +1,16 @@
+package org.infinispan.lock.configuration;
+
+/**
+ * Locks are stored in a container that can privilege availability or consistency.
+ * Availability is useful when you wish to be able to use the lock most of the time.
```

Comment:
I'm pretty sure everyone wants to be able to use the lock **at least** most of the time ;)

I would say something like

```
Locks are stored in a container that can privilege availability or consistency.

Most of the time, locks are both available and consistent. 
But in some situations, e.g. when the cluster splits, there is a choice between keeping the locks available everywhere (potentially allowing multiple nodes to acquire the same lock)
or making it unavailable in the minority partition(s) (potentially requiring administrator intervention to become available again).
@see org.infinispan.partitionhandling.PartitionHandling
```

Reply:
Ok! 😄